velox/exec/tests/SimpleAggregateAdapterTest.cpp

```diff
@@ -272,6 +272,63 @@ TEST_F(SimpleArrayAggAggregationTest, nestedArray) {
       {expected});
 }
 
+TEST_F(SimpleArrayAggAggregationTest, trackRowSize) {
+  core::QueryConfig queryConfig({});
+  auto testTractRowSize = [&](core::AggregationNode::Step step,
+  const VectorPtr& input,
+  bool testGlobal) {
+    auto fn = Aggregate::create(
+      "simple_array_agg",
+      step,
+      isRawInput(step) ? std::vector<TypePtr>{BIGINT()} : std::vector<TypePtr>{ARRAY(BIGINT())},
+      ARRAY(BIGINT()),
+      queryConfig);
+
+  HashStringAllocator stringAllocator{pool()};
+  memory::AllocationPool allocationPool{pool()};
+  fn->setAllocator(&stringAllocator);
+
+  int32_t rowSizeOffset = bits::nbytes(1);
+  int32_t offset = rowSizeOffset;
+  offset = bits::roundUp(offset, fn->accumulatorAlignmentSize());
+  fn->setOffsets(
+      offset,
+      RowContainer::nullByte(0),
+      RowContainer::nullMask(0),
+      rowSizeOffset);
+
+  auto size = input->size();
+  std::vector<char> group(offset + fn->accumulatorFixedWidthSize());
+  std::vector<char*> groups(size, group.data());
+  std::vector<vector_size_t> indices(size, 0);
+  fn->initializeNewGroups(groups.data(), indices);
+
+  if (isRawInput(step)) {
+    if (testGlobal) {
+        fn->addSingleGroupRawInput(group.data(), SelectivityVector(size), {input}, false);
```

Comment:
nit: maybe use temp variable for SelectivityVector(size) to make the code a bit easier to read